draft-ietf-httpbis-proxy-status.md

```diff
@@ -55,11 +55,13 @@ proxies") -- have become an increasingly significant part of HTTP deployments. I
 reverse proxies and Content Delivery Networks (CDNs) form part of the critical infrastructure of
 many Web sites.
 
-Typically, HTTP intermediaries forward requests towards the origin server and then forward their responses back to clients. However, if an error occurs, the response is generated by the intermediary itself.
+It is sometimes useful to append information to messages about intermediaries' operation, without changing the message content itself. For example, it's helpful to know details of their configuration and operational conditions for debugging purposes. To enable that, {{info-header}} defines a new HTTP header field to convey that information.
+
+Additionally, in some cases responses are generated by the intermediary itself, rather than being forwarded from the origin server. For example, this can happen when there is a configuration error, or there is a problem in communicating with the origin server.
 
 HTTP accommodates these types of errors with a few status codes; for example, 502 Bad Gateway and 504 Gateway Timeout. However, experience has shown that more information is necessary to aid debugging and communicate what's happened to the client.
 
-To address this, {{header}} defines a new HTTP response header field to convey such information, using the Proxy Status Types defined in {{types}}. {{register}} explains how to define new Proxy Status Types.
+To address this, {{status-header}} defines a new HTTP response header field to explicitly indicate that the response was generated by an intermediary, not the origin server, and to convey additional details using the Proxy Status Types defined in {{status-types}}. {{register-status}} explains how to define new Proxy Status Types.
```

Comment:
That isn't always true, though. e.g. `Proxy-Status: http_response_incomplete` could be sent in a trailer of an incomplete response.